src/config/schemes/old/fixreputationallocationparams.json

```diff
@@ -1,3 +1,4 @@
 {
-  "reputationReward" : 100
-}
+  "reputationReward": 100,
```

Comment:
The idea of keeping the `old` dir is just temporarily while we create all the missing migrations. 

Most of them can be simplified. For example I think in this case, we can use `src/conf/rep/initalRepDistribution`, it has already the values for how much REP we allocate per method and it's easier to mantain.